src/campsite-info/components/camp-rating/camp-rating.component.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box, Rating } from '@mui/material';
+import StarIcon from '@mui/icons-material/Star'
+import { ratingStyles, tooltipStyles } from './camp-rating.styles';
+
+const tooltipLabels: { [index: string]: string} = {
+  1: "Terrible",
+  2: "Bad",
+  3: "Average",
+  4: "Good",
+  5: "Awesome!",
+};
+
+function getLabelText(value: number) {
+  return `${value} Star${value !== 1 ? 's' : ''}, ${tooltipLabels[value]}`;
+}
+
+export const StarRating = () => {
+  const [value, setValue] = React.useState<number | null>(0); //this line shows the default rating
```

Comment:
This inline comment isn't super helpful. It's an idiomatic usage of useState and doesn't require documentation.

Reply:
alrighty, I removed the comment